lms/djangoapps/courseware/views/views.py

```diff
@@ -878,8 +878,10 @@ def course_about(request, course_id):
             'reviews_fragment_view': reviews_fragment_view,
             'sidebar_html_enabled': sidebar_html_enabled,
         }
-
-        return render_to_response('courseware/course_about.html', context)
+        if not course.product:
```

Comment:
@dyudyunov please, avoid boolean negation where possible. You'd better use `if course.product:` and refactor the below code accordingly.